tree/ntuple/v7/test/ntuple_modelext.cxx

```diff
@@ -204,8 +204,14 @@ TEST(RNTuple, ModelExtensionProject)
    {
       auto model = RNTupleModel::Create();
       auto fieldPt = model->MakeField<float>("pt", 42.0);
-
-      auto ntuple = RNTupleWriter::Recreate(std::move(model), "myNTuple", fileGuard.GetPath());
+      model->AddProjectedField(std::make_unique<RField<float>>("aliasPt"), [](const std::string &) { return "pt"; });
+      model->AddProjectedField(std::make_unique<RField<float>>("al2Pt"), [](const std::string &) { return "pt"; });
+      auto fieldE = model->MakeField<float>("E", 1.0);
+      model->AddProjectedField(std::make_unique<RField<float>>("aliasE"), [](const std::string &) { return "E"; });
+
+      RNTupleWriteOptions options;
+      options.SetUseBufferedWrite(false);
```

Comment:
This disables buffered writing, probably not to trigger the bug fixed in the first commit? We should probably extend our unit testing coverage for late schema extension...

Reply:
Thanks! I forgot to remove it... done now.